api/src/models.d.ts

```diff
@@ -72,3 +72,158 @@ export interface ParticipantActivityForProgram {
   program_id: number;
   participant_activities: ParticipantActivityCompletionStatus[];
 }
+
+export interface Question {
+  assessment_question_id: number;
+  id?: number;
+  assessment_id?: number;
+  title: string;
+  description?: string;
+  question_type: string;
+  answers?: Answer[];
```

Comment:
`Answer` here is an object type, one that we also define in this file. As per our feedback earlier today, I would like you to have the interface definition for `Answer` appear above `Question` in this file; likewise, any other object types being referenced in any of the other interfaces should appear above them in this file.